### PR TITLE
EWL-7076: Menu doesn't expand on people listing pages

### PR DESCRIPTION
**Pull request by russom-woldezghi**
## Ticket(s)
**Jira Ticket**
- [EWL-7076: Menu doesn't expand on people listing pages](https://issues.ama-assn.org/browse/EWL-7076)

## Description
- Fixes main menu behavior to dropdown/appear when user clicks on white X in purple main menu on people listing pages
  - http://ama-one.local/about/board-trustees/board-trustees-members
  - http://ama-one.local/member-groups-sections/medical-students/medical-student-section-mss-governing-council
- Hides scrollbar on Firefox/Safari browser (after clicking on X to expand main menu)
- See production to see bug
  - https://www.ama-assn.org/member-groups-sections/medical-students/medical-student-section-mss-governing-council
  - https://www.ama-assn.org/about/board-trustees/board-trustees-members 

## To Test
- [ ] Pull down code and enable local styleguide
- [ ] Navigate to the following pages to test main menu dropdown behavior: 
  - http://ama-one.local/about/board-trustees/board-trustees-members
  - http://ama-one.local/member-groups-sections/medical-students/medical-student-section-mss-governing-council 
- [ ] Click on X to expand main menu
- [ ] Ensure on other pages (any page) the main menu are not affected by fix.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-7076-menu-people-listing/html_report/index.html).

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/05-pages/_people-listing.scss [hunk 1]
```diff
@@ -1,7 +1,4 @@
 .ama__people-listing {
-  .container {
```
Comment (designerneil) [inline styleguide/source/assets/scss/05-pages/_people-listing.scss:2]: Why was this removed? The menu seems to work fine without this removal

Comment (russom-woldezghi) [reply styleguide/source/assets/scss/05-pages/_people-listing.scss:2]: @designerneil The `overflow: hidden;` attribute was causing the menu to not show. I was only able to have it expanded by removing the attribute. We can leave the the `.container` there, I don't think it will affect the menu. 

Also, is there a better way than removing `overflow: hidden;`? Seeing if there was a better way, in case of regression issues/bugs.

Comment (designerneil) [reply styleguide/source/assets/scss/05-pages/_people-listing.scss:2]: I think the overflow hidden was put in place to prevent the menu from going over the container. I'll check to see if there is a better way.

**Review by designerneil — APPROVED** [review]
